api/GovStack_Registration_BB_API-registration-0.0.1-resolved.json

```
@@ -0,0 +1,163 @@
+		"description": "Information Mediator BB Security server"
+	}],
+
+	"paths": {
+
+
+		"/api/process-definition/{key}/start-with-form": {
+			"post": {
+				"tags": ["application-registration"],
+				"summary": "Start application process instance with form",
+				"description": "Creates application process instance",
+				"operationId": "createProcessInstanceWithForm",
+				"parameters": [{
+						"$ref": "#/components/parameters/Information-Mediator-Client"
+					},
+
+					{
+						"name": "key",
+						"description": "I think this key is issued by the creator app and = id, ",
+						"in": "path",
+						"required": true,
+						"schema": {
+							"type": "string"
+						}
+					}
+				],
+				"requestBody": {
+					"content": {
+						"application/json": {
+							"schema": {
+								"$ref": "#/components/schemas/FormDataDto"
+							}
+						}
+					},
+					"required": true
+				},
+				"responses": {
+					"500": {
+						"description": "Internal server error",
+						"content": {
+							"*/*": {
+								"schema": {
+									"$ref": "#/components/schemas/SystemErrorDto"
+								}
+							}
+						}
+					},
+					"200": {
+						"description": "Returns started application process instance",
+						"content": {
+							"*/*": {
+								"schema": {
+									"$ref": "#/components/schemas/StartProcessInstanceResponse"
+								}
+							}
+						}
+					},
+					"404": {
+						"description": "Business process definition was not found",
+						"content": {
+							"*/*": {
+								"schema": {
+									"$ref": "#/components/schemas/SystemErrorDto"
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+
+
+
+
+
+	},
+	"components": {
+		"parameters": {
+			"Information-Mediator-Client": {
+				"in": "header",
+				"name": "Information-Mediator-Client",
+				"example": "eGovStack/GOV/90000009/digitalregistries",
+				"description": "Format is: INSTANCE/CLASS/MEMBER/SUBSYSTEM",
+				"schema": {
+					"type": "string"
+				},
+				"required": true
+			}
+		},
+		"schemas": {
+			"FormDataDto": {
+				"type": "object",
+				"properties": {
+					"data": {
+						"type": "object",
+						"additionalProperties": {
+							"type": "object"
+						}
+					},
+					"x-access-token": {
+						"type": "string"
+					}
+				}
+			},
+
+			"StartProcessInstanceResponse": {
+				"type": "object",
+				"properties": {
+					"id": {
+						"type": "string",
+						"description": "81c4445c-bff6-11ed-afa1-0242ac120002"
+					},
+					"processDefinitionId": {
+						"type": "string",
+						"description": "81c4445c-bff6-11ed-afa1-0242ac120002"
+					},
+					"ended": {
+						"type": "boolean",
+						"description": "true"
+					}
+				}
+			},
+			"SystemErrorDto": {
+				"type": "object",
+				"properties": {
+					"traceId": {
+						"type": "string",
+						"description": "81c4445c-bff6-11ed-afa1-0242ac120002"
+					},
+					"code": {
+						"type": "string",
+						"description": "5000"
```

```suggestion
						"description": "500"
```
this should probably be HTTP Status code 500?